### ISSUE TITLE
Codewind / PFE fails to load on Kube

### ISSUE THREAD

**Issue by johnmcollier** [eclipse/codewind#104]
I'm getting the following error in PFE after applying the fix for https://github.com/eclipse/codewind/issues/71 (https://github.com/eclipse/codewind-che-plugin/pull/47)

```
```[02/08/19 15:05:49 /portal/routes/environment.route.js] [ERROR] TypeError: Cannot read property 'spec' of undefined
    at getTektonDashboard (/portal/routes/environment.route.js:34:55)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[02/08/19 15:05:53 /portal/routes/environment.route.js] [ERROR] TypeError: Cannot read property 'spec' of undefined
    at getTektonDashboard (/portal/routes/environment.route.js:34:55)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[02/08/19 15:05:53 /portal/routes/environment.route.js] [ERROR] TypeError: Cannot read property 'spec' of undefined
    at getTektonDashboard (/portal/routes/environment.route.js:34:55)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[02/08/19 15:05:53 /portal/routes/environment.route.js] [ERROR] TypeError: Cannot read property 'spec' of undefined
    at getTektonDashboard (/portal/routes/environment.route.js:34:55)
    at process._tickCallback (internal/process/next_tick.js:68:7)```
```

**Comment by tetchel** [comment]
since no one is working today i will have a look at this

**Comment by tetchel** [comment]
this happens if tekton is not installed. tekton installation should not be mandatory. i will put in a fix

**Comment by tetchel** [comment]
fixed in #110
